static-site/app/groups/[[...groups]]/individual-group-page.tsx

```diff
@@ -247,6 +227,46 @@ export default function IndividualGroupPage({
   }
 }
 
+// It is unfortunate that this method repeats the request that we're already
+// making above in the `useEffect()` hook. Ideally we could make that request
+// once, and re-use the response.
+//   - jsja, 08 Aug 2025
+//   - copied/modified by victor, 21 Nov 2025
```

Comment:
I'm confused by this comment. I'd think we could keep the `datasets` state and pass that into `_resourcesCallback` to be used to create the `Group` object (something like https://github.com/nextstrain/nextstrain.org/commit/69073ccf1c612aed444d2436bdb4ffdbeb871935). Am I missing something here?